src/plugins/explore/public/components/visualizations/area/area_vis_config.ts

```diff
@@ -28,13 +30,15 @@ export interface AreaChartStyleControls {
   tooltipOptions: TooltipOptions;
 
   // Threshold and grid
-  thresholdLines: ThresholdLines;
+  thresholdLines?: ThresholdLines;
```

Comment:
Can you add a comment for all deprecated options:
```suggestion
  /**
   * @deprecated
   */
  thresholdLines?: ThresholdLines;
```